kubeval/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM alpine
+
+ENV KUBEVAL_VERSION=0.14.0 \
+    KUBECTL_VERSION=1.16.2
+
+RUN wget https://github.com/instrumenta/kubeval/releases/download/0.14.0/kubeval-linux-amd64.tar.gz && \
+    tar xf kubeval-linux-amd64.tar.gz && \
+    mv kubeval /usr/local/bin && \
```

Comment:
This would also need `chmod +x` I think 🤔

Reply:
It seems to work without.

Reply:
I'd probably add it anyways? Just in case? What do you think?
Sorry for asking for so much stuff 😅

Reply:
OK, why not.